### ISSUE TITLE
wavy visual effect when using the dock

### ISSUE THREAD

**Issue by mpatour** [Mazamars312/Analogue-Amiga#24]
Great work! But is it possible to remove the light "wavy" visual effect when using the core with the Analogue Dock?
Thanks

**Comment by anamegoeshere23** [comment]
Seconded.  But if it's intentional - a toggle option would be great

**Comment by soctib** [comment]
frame the README.md:

``` 
There is a bug being addressed by Analogue on the output at this moment and we hope they can report back on what is causing the video to move over on random lines.
```
and
```
[What will be the next update?](https://github.com/Mazamars312/Analogue-Amiga#what-will-be-the-next-update)
Fix up the USB mouse control once we find out what is happening here

Clock input for the Amiga to keep its time.

Fix up the video output on the dock - This looks to be a bug in the Analogue APF on sending the Framebuffer out the HDMI port. They are aware and confirming details for me.
```
So it's not intentional. However, if we have to wait for Analaogue to fix it that can take a long time... ☹️

**Comment by rsn8887** [comment]
This is fixed with Analogue OS 2.0

**Comment by limi** [comment]
Nice!
